quic/s2n-quic-platform/src/io/testing.rs

```diff
@@ -23,6 +23,11 @@ pub use time::now;
 
 pub use bach::task::{self, primary, spawn};
 
+// returns `true` if the caller is being executed in a testing environment
+pub fn is_in_env() -> bool {
```

Comment:
not sure I understand the name, why not something like `is_testing_env`?

Reply:
i was thinking `testing` was already in the module path name (`io::testing::is_in_env()`) so it seemed a bit redundant. But I don't have any strong opinions on the name.

Reply:
ah, yeah thats fine then